### DISCUSSION TITLE
how to list all export functions of a JS module? How to call the export function?

### DISCUSSION THREAD

**Discussion by i18nsite** [boa-dev/boa#3855] [Q&A]
how to list all export functions of a JS module? How to call the export function?

for example
```
export const abc=(a,b)=>{return a+b}
export default (a,b)=>{
    return a-b
  }
```

**Comment by jedel1043** [comment]
The returned namespace object is an ordinary object that contains all the properties you need. If you follow the [modules example](https://github.com/boa-dev/boa/blob/main/examples/src/bin/modules.rs) you can see how it accesses the object returned by `.namespace()` to extract the `mix` function from the module.

If you want to list all export functions, just call `own_property_keys` on the same object and that should give you the full list of all exports.
